examples/helm/kanister/kanister-mysql/README.md

```diff
@@ -20,7 +20,12 @@ For basic installation, you can install using the provided Helm chart that will
 Prior to install you will need to have the Kanister Helm repository added to your local setup.
 
 ```bash
-$ helm repo add kanister http://charts.kanister.io
+$ helm repo add kanister https://charts.kanister.io
+```
+You can use following command to install Kanister if not installed already
+
+```bash
+$ helm install --name kanister --namespace kasten-io kanister/kanister-operator --set image.tag=0.20.0
```

Comment:
[minor] Let's make this namespace `kanister`

Reply:
@pavannd1 will the e2e pass if we change this to ` kanister ` ?

Reply:
@pavannd1 in the latest commit in values.yaml, we are changing default namespace to `kanister-io` https://github.com/kanisterio/kanister/commit/5e5a18eae9a63f85060a8816bf07577c18bba88f